docs/src/rules/constructor-super.md

```diff
@@ -14,6 +14,16 @@ This rule checks whether or not there is a valid `super()` call.
 
 This rule is aimed to flag invalid/missing `super()` calls.
 
+This is a `SyntaxError`:
```

Comment:
It's a syntax error from the parser, so it doesn't throw an actual `SyntaxError` object.
```suggestion
This is a syntax error because there is no `extends` clause in the class:
```